tests/snapshots/tests.contrib.langchain.test_langchain_community.test_faiss_vectorstore_retrieval.json

```diff
@@ -10,14 +10,14 @@
     "error": 0,
```

Comment:
I'm seeing faiss added as a dependency and faiss snapshot tests, but not seeing them in the actual test file as tests. Can we either add them in, or remove these additional files / dependencies?